### PR TITLE
Rename "Replace" conflict option to "Merge" for folder uploads

### PR DESCRIPTION
**Pull request by AlexAndBear**
## Related Issue

- Fixes https://github.com/owncloud/web/issues/10640

## How Has This Been Tested?

- test environment:
- test case 1:
- test case 2:
- ...

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests
- [ ] Documentation
- [ ] Maintenance (e.g. dependency updates or tooling)

## Open tasks:

- [ ] ...

### CODE REVIEW

**Comment by AlexAndBear** [comment]
@JammingBen something like that, or am I getting it wrong?

File: packages/web-pkg/src/helpers/resource/conflictHandling/conflictDialog.ts [hunk 1]
```diff
@@ -27,7 +27,8 @@ export class ConflictDialog {
         [key: string]: string
       },
       disableHtmlEscaping?: boolean
-    ) => string
+    ) => string,
+    protected isUpload?: boolean
```
Comment (JammingBen) [inline packages/web-pkg/src/helpers/resource/conflictHandling/conflictDialog.ts:31]: Left-over from the previous approach I believe?

Comment (AlexAndBear) [reply packages/web-pkg/src/helpers/resource/conflictHandling/conflictDialog.ts:31]: Exactly

**Review by JammingBen — APPROVED** [review]
👍

**Pull request merged**